packages/core/src/telemetry/emitter.ts

```diff
@@ -113,8 +136,9 @@ export const telemetryEmitter = async ({ dataDir, version, debug }: {
       pushEvent({
         event,
         timestamp: new Date(),
-        distinct_id: currentId,
+        distinct_id: machineId,
```

Comment:
I think this change might disable [event linking](https://posthog.com/docs/getting-started/identify-users#linking-past-events-with-our-new-user). this is why it was using a mutable `currentId` which starts with an anonymous ID and later set to the profile at `identify`

Reply:
A profile can also have several users, so by using deviceId and profile as group, it's simplify the process of understanding how each user in a profile interact with preevy. Currently, switching a profile is linking your identity and it's a bit weird. In addition, using a session ID as an identifier creates too much noise.

As for linking:
It basically means that there's no linking for now (unless in the future we'll have a user identifier instead of a device identifier).